src/test/java/com/google/sps/BookmarkPlaceGuideServletTest.java

```diff
@@ -0,0 +1 @@
+
```

Comment:
Why do you want to merge to master an empty file?

Reply:
removed the file.